File: sentimental-cash/cash.py
from cs50 import get_float

while True:
    cash = get_float("Change: ")
    if (cash < 0):
        pass
    else:
        break

count = 0
change = cash * 100
while change > 0:

    if change >= 25:
        change = change - 25
        count += 1

    elif change >= 10:
        change = change - 10
        count += 1

    elif change >= 5:
        change = change - 5
        count += 1

    elif change >= 1:
        change = change - 1
        count += 1

print(count)

File: cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int count = 0;
    int cash = -1;
    while (cash < 0)
    {
        cash = get_int("Enter the change in cents\t");
    }
    while (cash != 0)
    {
        if (cash == 1)
        {
            cash = cash - 1;
            count++;
        }

        else if (cash > 1 && cash < 5)
        {
            cash = cash - 1;
            count++;
        }

        else if (cash >= 5 && cash < 10)
        {
            cash = cash - 5;
            count++;
        }

        else if (cash >= 10 && cash < 25)
        {
            cash = cash - 10;
            count++;
        }

        else if (cash >= 25)
        {
            cash = cash - 25;
            count++;
        }
    }
    printf("%d\n", count);
}
